### ISSUE TITLE
Conference Admin Dashboard

### ISSUE THREAD

**Issue by willykaram** [NYC-Camp/website#258]
We need a dashboard where we can get quick access to a bunch of administrative tools. So I've created a simple starting point for use to use here:
- http://www.nyccamp.org/admin/dashboard/conference

For now I've added a redirect from the /admin/dashboard to the above page. I'll add some theming and improve it over the weekend while doing some other theming, but for now I just wanted to put something basic in place that would hopefully be helpful immediately and that we could improve.

Hopefully nobody objects to this. But if so, we can disable and reverse/remove this if needed.

**Comment by likewhoa** [comment]
good starting point but we should focus on making everything mobile friendly before it's final.

**Comment by willykaram** [comment]
@likewhoa of course, right now this is pretty awful. It's just aimed at helping us get through the next week as stop-gap measure.
